### DISCUSSION TITLE
Spalling should deal bleeding damage.

### DISCUSSION THREAD

**Discussion by girlyguppy** [FakeFishGames/Barotrauma#15432] [Ideas and suggestions]
Large lacerations all over your body from giant chunks of metal flying at you? Just use a single morphine lol, easy fix, no danger 

They should cause bleed. It would make abyss creatures more threatening and it would make sense.

Perhaps spalling should even pierce through some of the interior hull, as interior rooms are completely immune to large creature damage currently.

**Comment by SomeRandomNoobKekeke** [comment]
Honest spallings reaction:
![spallings](https://github.com/user-attachments/assets/f7a2e511-bb44-41ee-9ec4-988f9c1749d6)
